robot/web/web.go

```diff
@@ -365,42 +364,43 @@ func (svc *webService) updateResources(resources map[resource.Name]resource.Reso
 		groupedResources[n.API] = r
 	}
 
-	apiRegs := resource.RegisteredAPIs()
-	for a, v := range groupedResources {
```

Comment:
definitely a chance for some existing resources to no longer exist and not appear in this list, preventing this function from updating the collection correctly